src/app/globals.css

```
@@ -1,27 +1,11 @@
+@tailwind utilities;
 
-button svg[data-noinherit="true"] {
-    stroke: unset !important;
-    fill: unset !important;
-    color: unset !important;
-}
 .filter-white {
     filter: brightness(0) invert(1);
 }
+
+@import "../style/fonts.css";
```

Biome (2.1.2)

[error] This @import is in the wrong position.

Any @import rules must precede all other valid at-rules and style rules in a stylesheet (ignoring @charset and @layer), or else the @import rule is invalid.
Consider moving import position.

(lint/correctness/noInvalidPositionAtImportRule)